mono/metadata/threads.c

```diff
@@ -180,7 +181,7 @@ static MonoThreadCleanupFunc mono_thread_cleanup_fn = NULL;
 
 /* The default stack size for each thread */
 static guint32 default_stacksize = 0;
-#define default_stacksize_for_thread(thread) ((thread)->stack_size? (thread)->stack_size: default_stacksize)
```

Comment:
Line damage.

Reply:
Broken macros. They need parens. But can be a separate PR. I'll dot that.